package.json

```diff
@@ -86,6 +86,7 @@
     "@typechain/web3-v1": "^1.0.0",
     "@types/chai": "^4.2.11",
     "@types/mocha": "^7.0.2",
+    "@types/bn.js": "^4.11.6",
```

Comment:
AFAIK, this shouldn't be needed since typings are included in `bn.js` module already.